spec/poros/food_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Object' do
+    it 'can read the Food object attributes' do
+        food = FoodFacade.search_foods("sweet potatoes")
+        first_food_item = food.first
+
+        expect(first_food_item.brand_owner).to eq("ARCHER FARMS")
+        expect(first_food_item.code).to eq("492111402857")
+        expect(first_food_item.description).to eq("SWEET POTATOES")
+        expect(first_food_item.ingredients).to eq("SWEET POTATOES.")
+    end
```

Comment:
No need to call to your facade in this test since you are just testing the poro. If you could mock out the data that would get passed to the poro, you could do something like `food = Food.new(mocked_food_json_data)`, and then test that object.